packages/lib/src/vis/rgb/utils.ts

```diff
@@ -1,10 +1,12 @@
 import { createArrayFromView } from '@h5web/shared';
-import ndarray from 'ndarray';
+import type { NdArray, TypedArray } from 'ndarray';
 
-export function flipLastDimension(value: number[], dims: number[]) {
-  const baseArray = ndarray(value, dims);
```

Comment:
No longer converting to `ndarray` here so I can call `toTypedNdArray` in `RgbVis` directly (and skip the conversion if the ndarray is already typed).

Reply:
Great idea :+1: